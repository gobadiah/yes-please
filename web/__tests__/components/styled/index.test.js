import React from 'react';
import renderer from 'react-test-renderer';

import {
  Flex,
  FlexColumn,
} from '../../../components/styled';

describe('Styled components', () => {
  describe('Flex', () => {
    it('should match snapshot', () => {
      const component = renderer.create(<Flex />);
      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });
  });

  describe('FlexColumn', () => {
    it('should match snapshot', () => {
      const component = renderer.create(<FlexColumn />);
      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
