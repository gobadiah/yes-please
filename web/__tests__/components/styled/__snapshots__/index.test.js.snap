// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled components Flex should match snapshot 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div
  className="emotion-0 emotion-1"
/>
`;

exports[`Styled components FlexColumn should match snapshot 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  className="emotion-0 emotion-1"
/>
`;
